portfolio/src/main/webapp/style.css

```diff
@@ -1,28 +1,29 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 1000px;
+  width: 75%;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
 
-#img-format{
+.img-format{
     display: block;
     text-align: center;
     margin-left: auto;       
     margin-right: auto;
-    max-width: 100%;
+    max-width: 70%;
     height: auto;
+    object-fit: cover;
 }
 
-#img-container{
-    display: block;
-    text-align: center;
+.img{
+    width: 75%;
+    height: auto;
     object-fit: cover;
 }
 
-#center{
+.center{
     text-align: center;
 }
```

Comment:
Indentation should be consistent throughout the file; internally we use two spaces to indent, so I'd recommend that here.